lib/assets/javascripts/init/tinymce.js

```diff
@@ -84,11 +84,11 @@ $(document).ready(function() {
     toolbar1: "undo redo | bold italic underline strikethrough |  alignleft aligncenter alignright alignjustify | styleselect | fontsizeselect",
     toolbar2: "forecolor backcolor | bullist numlist outdent indent | link unlink anchor image media | removeformat | code preview fullscreen",
     language: $("html").attr("lang"),
-    extended_valid_elements : "div[*],span[*],iframe[src|width|height|name|align|frameborder|scrolling|id|onload],applet[code|codebase|archive|name|id|width|height|param],video[*],source[*],audio[*]",
+    extended_valid_elements: "div[*],span[*],iframe[src|width|height|name|align|frameborder|scrolling|id|onload],applet[code|codebase|archive|name|id|width|height|param],video[*],source[*],audio[*]",
```

Comment:
Line is too long.<br>Identifier 'extended_valid_elements' is not in camel case.<br>Mixed double and single quotes.